src/App.tsx

```diff
@@ -15,47 +16,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+type Sorts = 'SORT_ALPHABETICALLY' | 'SORT_BY_LENGTH' | null;
```

Comment:
Requirements: `Use enum SortType instead of string literals for possible sorting options (including the default one)`